pkg/apply/gen.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"github.com/labring/sealos/pkg/apply/processor"
 	"github.com/labring/sealos/pkg/buildah"
-	"github.com/labring/sealos/pkg/runtime"
+	runtime "github.com/labring/sealos/pkg/runtime/kubernetes"
```

Comment:
Maybe use NewRuntime(runtime type) is better than use kuberntes package directly.